src/main/java/org/folio/circulation/domain/UpdateRequestQueue.java

```diff
@@ -61,46 +63,86 @@ public static UpdateRequestQueue using(Clients clients) {
     RequestQueue requestQueue, String checkInServicePointId) {
 
     if (requestQueue.hasOutstandingFulfillableRequests()) {
-      Request firstRequest = requestQueue.getHighestPriorityFulfillableRequest();
+      return updateOutstandingRequestOnCheckIn(requestQueue, checkInServicePointId);
+    } else {
+      return completedFuture(succeeded(requestQueue));
+    }
+  }
+
+  private CompletableFuture<Result<RequestQueue>> updateOutstandingRequestOnCheckIn(
+    RequestQueue requestQueue, String checkInServicePointId) {
+
+    Request req = requestQueue.getHighestPriorityFulfillableRequest();
```

Comment:
Why is the name of this variable abbreviated?

```suggestion
    Request request = requestQueue.getHighestPriorityFulfillableRequest();
```
```suggestion
    Request requestBeingFulfilled = requestQueue.getHighestPriorityFulfillableRequest();
```